birdhouse/README.rst

```diff
@@ -19,17 +19,17 @@ Requirements:
   Install latest docker-ce and docker-compose for the chosen distro (not the
   version from the distro).
 
-To run ``docker-compose`` for PAVICS, the `pavics-compose.sh <birdhouse/pavics-compose.sh>`_ wrapper script must be used.
+To run ``docker-compose`` for PAVICS, the `pavics-compose.sh <pavics-compose.sh>`_ (:download:`download </birdhouse/pavics-compose.sh>`) wrapper script must be used.
```

Comment:
Those `:download:download </birdhouse/pavics-compose.sh>` are actually showing up when browsing on GH.

Reply:
Hum probably we have to live with seeing those download instruction when browsing GH.

Reply:
Instead of using the :download role, maybe the :doc role?  https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role (I have no idea what I am saying, I never used it, just found by googling).

Reply:
The `:doc:` approach only works for linking to rendered pages. This doesn't work for things that need to be presented in their raw formats (shell scripts, YAML, JSON, etc.).

The alternative is to simply provide the GitHub address to the file and perform a link-check on build. The downside is that if you add or delete a file being pointed to, the docs will need to be pushed to master before you know they work (similar to the issues with adding new PyWPS services that need to be tested online). This could also be done?